velox/tpch/gen/dbgen/include/dbgen/dss.h

```diff
@@ -435,9 +435,21 @@ int dbg_print(int dt, FILE* tgt, void* data, int len, int eol);
 #define PR_STRT(fp) /* any line prep for a record goes here */
 #define PR_END(fp) fprintf(fp, "\n") /* finish the record here */
 #ifdef MDY_DATE
-#define PR_DATE(tgt, yr, mn, dy) sprintf(tgt, "%02d-%02d-19%02d", mn, dy, yr)
+#define PR_DATE(tgt, yr, mn, dy)                                \
+  do {                                                          \
+    auto res = sprintf(tgt, "19%02ld-%02ld-%02ld", yr, mn, dy); \
+    if (FOLLY_UNLIKELY(res < 0)) {                              \
```

Comment:
This path is not performance sensitive. Remove FOLLY_UNLIKELY to avoid folly dependency.